src/applications/edu-benefits/feedback-tool/components/ListItem.jsx

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-telephone-component */
```

Comment:
I need to use the ESLint because it removes my changes when I do git commit in the following code lines:
From this code 
"<va-telephone
                contact={ele.value}
                tty={ele.tty}
                international={ele.international}
              />"
to this code:
              "<a href={`tel:${ele.value}`}>{ele.number}</a>"

I need the <a> but it changes it to <va-telephone>.